docs/Maintainer-Guidelines.md

```diff
@@ -80,6 +80,13 @@ Here’s a flowchart for managing a PR which is ready to merge:
 
 ![Flowchart for managing pull requests](assets/img/docs/managing-pull-requests.drawio.svg)
 
+Here are guidelines about when to use squash & merge versus rebase & merge. These options should only be used when bottles are not needed.
```

Comment:
```suggestion
Here are guidelines about when to use squash & merge versus rebase & merge. These options should only be used with PRs where bottles are not needed or affected.
```

As above, it may be beneficial to disambiguate the language around bottles here but there may be a better way of expressing this than what I've suggested.

Reply:
👍 Like the added clarity about needed vs affected